backend/maskgen_api/views.py

```diff
@@ -222,7 +232,44 @@ def view_list(self, request):
 
         return Response(results)
 
-    # TODO: delete obj
+    @action(detail=False, methods=["delete"], url_path="delete")
+    def delete_obj(self, request):
+        list_name = request.query_params.get("list_name")
+        obj_name = request.query_params.get("obj_name")
+        user_id = request.headers.get("user-id")
+        obj_list = get_object_or_404(ObjectList, name=list_name, user_id=user_id)
+        obj = obj_list.objects.get(name=obj_name)
+        obj_list.objects_list.remove(obj)
+        return Response(
+            {"message": f"object '{obj_name}' removed from list '{list_name}'"},
+            status=status.HTTP_200_OK,
+        )
+
+    @action(detail=False, methods=["patch"], url_path="edit")
```

Comment:
since we added the edit, we should not re-use objects across other object lists, otherwise, when you update an object in one object list, it will change on other lists and, possibly, other projects.